apps/web/src/components/submission/components/Review.tsx

```diff
@@ -22,6 +23,7 @@ import {
   deploymentDurationOptions,
   placementOptions as allPlacementOptions,
 } from '../validation/preferences';
+import { Alert } from '../../Alert';
```

Comment:
Last small comment, could you add alert to `components/index.ts` and import as `from '@components'`